src/main/scala/ai/privado/model/exporter/AndroidPermission.scala

```diff
@@ -9,10 +9,15 @@ case class AndroidPermissionModel(
   permissionDetail: AndroidPermissionDetailModel
 )
 
-case class AndroidPermissionDetailModel(sourceId: String, occurrence: DataFlowSubCategoryPathExcerptModel)
+case class AndroidPermissionDetailModel(sourceId: String, occurrences: List[DataFlowSubCategoryPathExcerptModel])
 
 object AndroidPermissionsEncoderDecoder {
 
+  implicit val dataFlowSubCategoryPathExcerptModelDecoder: Decoder[DataFlowSubCategoryPathExcerptModel] =
+    deriveDecoder[DataFlowSubCategoryPathExcerptModel]
+  implicit val dataFlowSubCategoryPathExcerptModelEncoder: Encoder[DataFlowSubCategoryPathExcerptModel] =
```

Comment:
These encoder decoder are allready present at `model/exporter/Dataflow.scala`, just import this at the place where encoding/decoding is required